launcher/README.md

```diff
@@ -0,0 +1,22 @@
+The preflight updater GUI currently supports both PyQt4 and PyQt5. To
+enforce the use of PyQt5, set the environment variable SDW_UPDATER_QT to 5.
+
+## Why support PyQt4 and PyQt5?
+
+Qubes 4.0.3 uses an end-of-life Fedora template in dom0 (fedora-25). See
+rationale here:
+
+https://www.qubes-os.org/doc/supported-versions/#note-on-dom0-and-eol
+
+fedora-25 only includes PyQt4, which is why we have to support it for now.
+PyQt4 itself is no longer maintained, and is best installed through system
+packages, e.g., https://packages.debian.org/buster/python3-pyqt4
+
+The next version of Qubes, Qubes 4.1, will include PyQt5 in dom0.
+
+## Installing PyQt5
+
+Activate a virtualenv and intall the requirements using the following
+command:
+
+pip install --require-hashes -r qt5-requirements.txt
```

Comment:
are these requirements strictly required here? In my dev vm (Debian 10) I can run the launcher with QT4 and QT5 outside a virtualenv with no dependencies, using system-level qt packages. Is there a reason we should treat qt5 as a special case and track requirements ?

Reply:
The main reason I think this may make sense is so that we all use a predictable version of PyQt during development, with a defined hash, which could be important during debugging. (For now it will fetch the same version as we use with securedrop-client, but ideally it should match whatever is shipped with Qubes 4.1.) This was not an option for PyQt4, which is no longer available via pypi.org, but it is an option for PyQt5.

We could consolidate test-requirements.txt and qt5-requirements.txt into a single dev-requirements.txt.

Reply:
Since we aren't shipping/packaging/locking the PyQT dependency in dom0, I don't see the value in locking for development environment consistency (especially since we are using OS level packages)
However, locking for testing consistency makes sense here (since it will run both on developer machines but also in CI). Adding it to test-requirements (and optionally renaming, no strong preference here) sounds like a good approach to me

Reply:
I've refactored accordingly. I cheated by not updating all requirements but only inserting the up-to-date Qt5 requirements into `dev-requirements.txt`. (Updating all requirements will pull in a new version of `black` that'll require formatting changes.) Once this PR lands I think it'd be good idea to do a requirements refresh for both preflight updater & main repo requirements.